checkbox-ng/plainbox/impl/session/state.py

```diff
@@ -815,6 +816,15 @@ def trim_job_list(self, qualifier):
                 self.on_job_removed(job)
                 self.on_unit_removed(job)
 
+    def update_system_information(self, system_information):
```

Comment:
This is done this way rather than by directly assigning the variable because I wanted to make it similar to the other properties. I'm not a fan of this pattern of having empty setters and getters in python but the code in this file seems to call for that.